components/ui/header.tsx

```diff
@@ -1,7 +1,22 @@
 import React from "react";
-
+import Button from "@/components/ui/button";
+import logo from "./assets/Logo.svg"
```

Comment:
Logo as well as other static assets will be placed in a folder called public in the root of the project. I suggest to place images in this path `/public/img/logo.svg`

Using this, ve avoid importing the image as a component, and we get rid of the unnecessary JS that will slow down our project in production 

Check this out to understand better about the thing I wrote above

https://nextjs.org/docs/app/building-your-application/optimizing/static-assets